phpcs.xml.dist

```diff
@@ -1,108 +1,30 @@
-	</rule>
-	<rule ref="Squiz.Commenting.FunctionComment.MissingParamTag">
-		<exclude-pattern>tests/*</exclude-pattern>
-	</rule>
-	<rule ref="Generic.Commenting.DocComment.Empty">
-    	<exclude-pattern>tests/*</exclude-pattern>
-    </rule>
-	<rule ref="Generic.Commenting.DocComment.MissingShort">
-		<exclude-pattern>tests/*</exclude-pattern>
-	</rule>
-	<rule ref="Squiz.Commenting.VariableComment.Missing">
-		<exclude-pattern>tests/*</exclude-pattern>
-	</rule>
-	<rule ref="Squiz.Commenting.FunctionCommentThrowTag.Missing">
-		<exclude-pattern>tests/*</exclude-pattern>
-	</rule>
-
-	<!-- Do not apply filename rules for unit tests and object cache -->
 	<rule ref="WordPress.Files.FileName.NotHyphenatedLowercase">
 		<exclude-pattern>server-timing/object-cache.copy.php</exclude-pattern>
-		<exclude-pattern>tests/*</exclude-pattern>
-	</rule>
-	<rule ref="WordPress.Files.FileName.InvalidClassFileName">
-		<exclude-pattern>tests/*</exclude-pattern>
-	</rule>
-
-	<!-- Do not apply compatibility rules to allow using the modern PHPUnit functionality -->
-	<rule ref="PHPCompatibility.FunctionDeclarations.NewReturnTypeDeclarations.stringFound">
-		<exclude-pattern>tests/utils/*</exclude-pattern>
-	</rule>
-	<rule ref="PHPCompatibility.FunctionDeclarations.NewReturnTypeDeclarations.boolFound">
-		<exclude-pattern>tests/utils/*</exclude-pattern>
 	</rule>
 
-	<!-- Ignore inapplicable WordPress-Extra sniffs in tests. -->
-	<rule ref="WordPress.WP.EnqueuedResourceParameters">
-		<exclude-pattern>tests/*</exclude-pattern>
-	</rule>
-	<rule ref="WordPress.WP.AlternativeFunctions.file_get_contents_file_get_contents">
-		<exclude-pattern>tests/*</exclude-pattern>
-	</rule>
-	<rule ref="WordPress.Security.EscapeOutput.OutputNotEscaped">
-		<exclude-pattern>tests/*</exclude-pattern>
-	</rule>
-	<rule ref="WordPress.WP.AlternativeFunctions.unlink_unlink">
-		<exclude-pattern>tests/*</exclude-pattern>
-	</rule>
-	<rule ref="WordPress.WP.GlobalVariablesOverride.Prohibited">
-		<exclude-pattern>tests/*</exclude-pattern>
-	</rule>
-	<rule ref="WordPress.PHP.DevelopmentFunctions.error_log_var_export">
-		<exclude-pattern>tests/*</exclude-pattern>
-	</rule>
+	<file>.</file>
+	<exclude-pattern>./build/*</exclude-pattern>
+	<exclude-pattern>./node_modules/*</exclude-pattern>
+	<exclude-pattern>./plugins/*</exclude-pattern>
+	<exclude-pattern>./vendor/*</exclude-pattern>
 
-	<!-- Exceptions for variable name casing. -->
-	<rule ref="WordPress.NamingConventions.ValidVariableName">
-		<properties>
-			<property name="allowed_custom_properties" type="array">
-				<element value="nodeValue"/>
-				<element value="parentNode"/>
-				<element value="createTextNode"/>
-				<element value="textContent"/>
-			</property>
-		</properties>
+	<!-- Exclude specific files from the TextDomainMismatch sniff -->
+	<rule ref="WordPress.WP.I18n.TextDomainMismatch">
+		<exclude-pattern>admin/load.php</exclude-pattern>
+		<exclude-pattern>admin/plugins.php</exclude-pattern>
 	</rule>
```

Comment:
I think it would be simpler and cleaner to keep allowing `default`, as we have so far. That's not prohibited by any means. See my other review feedback.